config/routes.rb

```diff
@@ -17,6 +17,8 @@
   get 'reports/present/:id', to: 'reports#present', as: 'present_report'
   get 'reports/preview/:id', to: 'reports#preview', as: 'preview_report'
 
+  get 'chatbot/:topic', to: 'chatbot#show'
```

Comment:
For now I would leave the reports out. We're implying the current report.

We should rethink the whole report structure and evaluate what it is actually good for.